### ISSUE TITLE
Please don't use the qurator namespace

### ISSUE THREAD

**Issue by mikegerber** [qurator-spk/eynollah#112]
Apparently there is still a namespace problem when installing e.g. eynollah and sbb_binarization together and I think it should be resolved by not using "qurator" as part of the namespace (move "eynollah/" one level down and fix all references accordingly)

I've checked sbb_binarization and I don't think it should not conflict anymore but I'll check tomorrow to make sure. (@bertsky reported this in #106)

❗ However IF it still is a problem, then this is a serious bug that needs to be fixed ASAP

**Comment by mikegerber** [comment]
Installing eynollah and sbb_binarization (both main/master) into one Python 3.9 venv does not cause immediately visible breakage. I can call both tools' CLIs' `--help`:

| Call | result |
|--|--|
| `eynollah --help`                        | works
| `ocrd-eynollah-segment --help` | works
| `sbb-binarize --help`                  | works
| `ocrd-sbb-binarize --help`          | works

Installing sbb_tools (master again) into the same venv also does not cause breakage with e.g. `altotool --help`, it seems. So I am somewhat questioning if there is still a problem but as didn't test the released versions this could happen for those.

What I did NOT test:
@bertsky mentioned installing in devel mode, and I didn't install it devel mode. This might be relevant in a way I don't see now. (I am also not recommending doing it, but that's not the point.)

What I am STILL saying:
Using the `qurator` namespace should be avoided. Not sharing a namespace makes life easier.

I don't think I have anything else to contribute here.

**Comment by mikegerber** [comment]
Ah, I see, this is because the revert by @bertsky is in main already. And there's not a problem in sbb_binarization anymore.

So this needs more investigation. 

TODO:
- [ ] revert the revert
- [ ] test again with sbb_tools
- [ ] verify that sbb_binarization really has no problem anymore.

**Comment by cneud** [comment]
Just a brief sidenote fyi - as of Eynollah 0.3.0, the model used by it for binarization is identical to the one of https://github.com/qurator-spk/sbb_binarization. As one of the next steps, binarization will be exposed as an invididual processing step in Eynollah, and eventually we will deprecate the separate https://github.com/qurator-spk/sbb_binarization project.

**Comment by neingeist** [comment]
> Just a brief sidenote fyi - as of Eynollah 0.3.0, the model used by it for binarization is identical to the one of https://github.com/qurator-spk/sbb_binarization. As one of the next steps, binarization will be exposed as an invididual processing step in Eynollah, and eventually we will deprecate the separate https://github.com/qurator-spk/sbb_binarization project.

I'd tentatively say that sbb_binarization is out of the picture anyway because it seems to be moved away from the qurator namespace.

→ Need to check the other projects in combinations.

**Comment by bertsky** [comment]
The namespace problem has ultimately been fixed by #133, so this should be closed.

(And #151 will bring a functional replacement version of ocrd-sbb-binarize.)

**Comment by kba** [comment]
Yup, fixed in 84b8442 and sbb_binarization was merged into eynollah as well.
